src/DataCollection.UWP/Views/IdentifiedFeaturePopup.xaml

```diff
@@ -1,15 +1,121 @@
+                                                <ColumnDefinition Width="40"/>
+                                            </Grid.ColumnDefinitions>
+                                            <TextBlock Text="{Binding RelatedTable.TableName, Mode=OneWay}" FontSize="17" Padding="10" Grid.Column="0"/>
+                                        </Grid>
+                                        <ItemsControl ItemsSource="{Binding OriginRelatedRecords,Mode=OneWay}" BorderThickness="0" HorizontalContentAlignment="Stretch" Margin="3">
+                                            <ItemsControl.ItemTemplate>
+                                                <DataTemplate>
+                                                    <Grid>
+                                                        <Grid.ColumnDefinitions>
+                                                            <ColumnDefinition Width="*"/>
+                                                            <ColumnDefinition Width="40"/>
+                                                        </Grid.ColumnDefinitions>
+                                                        <Grid.RowDefinitions>
+                                                            <RowDefinition Height="5"/>
+                                                            <RowDefinition Height="*"/>
+                                                        </Grid.RowDefinitions>
+                                                        <StackPanel Orientation="Vertical" Grid.Column="0" Grid.Row="1" x:Name="OriginRecordGrid"> 
+                                                            <TextBlock Text="{Binding DisplayedFields[0].Field.Label, Mode=OneWay}" FontSize="12" FontWeight="Bold" Margin="5,0,0,0"/>
+                                                            <TextBlock Text="{Binding DisplayedFields[0].FormattedValue, Mode=OneWay}" FontSize="12" Foreground="Gray" Margin="7,0,0,0"/>
+                                                            <TextBlock Text="{Binding DisplayedFields[1].Field.Label, Mode=OneWay}" FontSize="12" FontWeight="Bold" Margin="5,0,0,0"/>
+                                                            <TextBlock Text="{Binding DisplayedFields[1].FormattedValue, Mode=OneWay}" FontSize="12" Foreground="Gray" Margin="7,0,0,0"/>
+                                                            <TextBlock Text="{Binding DisplayedFields[2].Field.Label, Mode=OneWay}" FontSize="12" FontWeight="Bold" Margin="5,0,0,0"/>
+                                                            <TextBlock Text="{Binding DisplayedFields[2].FormattedValue, Mode=OneWay}" FontSize="12" Foreground="Gray" Margin="7,0,0,10"/>
+                                                        </StackPanel>
+                                                        <Button Content="&#xE76C;" FontFamily="Segoe MDL2 Assets" FontSize="15" Width="30" Height="30" VerticalAlignment="Center"
```

Comment:
I think these buttons aren't wide enough for the arrow at the current font size, at least on my device.  

![image](https://user-images.githubusercontent.com/29742178/54959902-dfdaa080-4f17-11e9-9769-50feb39ecd8d.png)

Reply:
I'm going to work on the "look" once the functionality in implemented. I don't want to mess with it until I have all the UI components in. Trying to avoid doing the work twice. At the end, it will look more or less like the WPF one does